modules/preferences/main/index.coffee

```diff
@@ -284,10 +284,8 @@ class Preferences extends hx.EventEmitter
   supportedTimezones: option 'supportedTimezones'
   timezoneOffsetLookup: option 'timezoneOffsetLookup'
 
-  applyTimezoneOffset: (date, offset) ->
-    offset ?= @_.timezoneOffsetLookup(@timezone(), date.getTime()) || 0
-    utc = date.getTime() + (date.getTimezoneOffset() * 60000)
-    new Date(utc + offset * 60 * 60 * 1000)
+  applyTimezoneOffset: (date) ->
+    new Date(date.getTime() + (date.getTimezoneOffset() * 60000))
```

Comment:
I'll try to do it this night. The problem is that when you change the time zone of your computer tests really do not work.